src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -80,20 +81,28 @@
                 {:content (drawers/reactions-and-actions message-data context)}]))
 
 (defn user-message-content
-  [{:keys [content-type quoted-message content] :as message-data}
-   {:keys [chat-id] :as context}]
+  [{:keys [content-type quoted-message content outgoing outgoing-status] :as message-data}
+   {:keys [chat-id] :as context}
+   show-delivery-state?]
   (let [context     (assoc context :on-long-press #(message-on-long-press message-data context))
         response-to (:response-to content)]
     [rn/touchable-highlight
      {:underlay-color (colors/theme-colors colors/neutral-5 colors/neutral-90)
-      :style          {:border-radius 16}
-      :on-press       #()
+      :style          {:border-radius 16
+                       :opacity       (if (or (not outgoing) (= outgoing-status :sent)) 1 0.5)}
+      :on-press       (fn []
+                        (when (and outgoing
+                                   (not (= outgoing-status :sending))
+                                   (not @show-delivery-state?))
+                          (reset! show-delivery-state? true)
+                          (js/setTimeout #(reset! show-delivery-state? false) 3000)))
       :on-long-press  (fn []
                         (rf/dispatch [:dismiss-keyboard])
                         (rf/dispatch [:bottom-sheet/show-sheet
                                       {:content (drawers/reactions-and-actions message-data
                                                                                context)}]))}
-     [rn/view {:padding-vertical 8}
+     [rn/view
+      {:padding-vertical 8}
```

Comment:
The style should be inside `{:style ...}`.

Reply:
Yes I don't know why there are some cases where style is set without `{:style ...}`, will replace the other cases in this file too

Reply:
Thanks!